commands/bmc.vegman

```diff
@@ -116,12 +116,17 @@ function cmd_datetime_method {
   method=${method##*.}
   method=${method%\"*}
 
+  # VEGMAN doesn't have RTC and in manual mode it uses PCH time.
+  if [ "${method}" = "Manual" ]; then
+      method="HostTime"
+  fi
+
   if [[ $# -eq 0 ]]; then
       echo "Current time synchronization method is ${method}"
   else
     case "${1}" in
-      [Mm][Aa][Nn][Uu][Aa][Ll])
-        if [[ ${method} != "Manual" ]]; then
+      [Hh][Oo][Ss][Tt][Tt][Ii][Mm][Ee])
```

Comment:
better use `${1,,}` and compare to a lowercase string.

Reply:
done